l10n_ar_withholding_ux/views/account_tax_view.xml

```diff
@@ -73,7 +73,7 @@
         <field name="res_model">account.tax</field>
         <field name="view_mode">tree,form</field>
         <field name="domain" eval="False"/>
-        <field name="context">{'search_default_customer': True, 'search_default_supplier': True, 'default_type_tax_use': 'supplier', 'active_test': False}</field>
```

Comment:
add other field with default supplier
and update the manifest

Reply:
https://github.com/ingadhoc/odoo-argentina/blob/279ee282fc86fdfee2b52bfad857c105178857ad/l10n_ar_account_withholding/models/account_tax.py#L26

this one also change it
ask to Filo